### PR TITLE
Jenkins mods2

### PR DESCRIPTION
**Pull request by bmesnet**
add code for jenkins tests  + display_traces for debugging

### CODE REVIEW

File: hardware/sim/testlist.sh [hunk 1]
```diff
@@ -469,24 +417,14 @@
       done
     fi # hls_nvme_memcopy
  #
-    if [[ "$t0l" == "10141008" || "${env_action}" == "hls_helloworld"* ]];then echo -e "$del\ntesting helloworld"
+    if [[ "$t0l" == "10143008" || "${env_action}" == "hls_helloworld"* ]];then echo -e "$del\ntesting helloworld"
```
Comment (luyong6) [inline hardware/sim/testlist.sh]: Do we still use this testlist.sh?

Comment (bmesnet) [reply hardware/sim/testlist.sh]: i guess so

File: hardware/sim/testlist.sh [hunk 2]
```diff
@@ -417,7 +417,7 @@
       done
     fi # hls_nvme_memcopy
  #
-    if [[ "$t0l" == "10143008" || "${env_action}" == "hls_helloworld"* ]];then echo -e "$del\ntesting helloworld"
+    if [[ "$t0l" == "10141008" || "${env_action}" == "hls_helloworld"* ]];then echo -e "$del\ntesting helloworld"
```
Comment (luyong6) [inline hardware/sim/testlist.sh]: 3008

File: hardware/setup/patch_version.sh [hunk 3]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 #
-# Copyright 2016, International Business Machines
+# Copyright 2010, International Business Machines
```
Comment (luyong6) [inline hardware/setup/patch_version.sh]: 2019

**Pull request merged**